scene/gui/graph_edit.cpp

```diff
@@ -760,6 +760,11 @@ bool GraphEdit::is_in_hot_zone(const Vector2 &pos, const Vector2 &p_mouse_pos) {
 			continue;
 		}
 		Rect2 rect = child->get_rect();
+
+		// To prevent intersections with other nodes.
+		rect.set_position(rect.position * zoom);
+		rect.set_size(rect.size * zoom);
```

Comment:
You could just do
```
Rect2 rect = child->get_rect() * zoom;
```

Reply:
That would be cool but Rect2 does not have such operator :/

Reply:
Ah, right. But
```
rect.position *= zoom
rect.size *= zoom
```
should be possible.

Reply:
Fixed.